rdr_service/tools/tool_libs/curation.py

```diff
@@ -702,6 +703,30 @@ def _select_participant_ids(self, session, origin:str, cutoff_date:datetime=None
         result = query.order_by(Participant.participantId).all()
         self.pid_list = [pid[0] for pid in result]
 
+    def _populate_death_table(self, session: sqlalchemy.orm.session):
+        # Populates death table from deceased_report
+        self._set_rdr_model_schema([DeceasedReport])
+        column_map = {
+            Death.id: literal("0"),
```

Comment:
The `id` column is a primary key and would throw an error if any more than one record is inserted. Should the `id` column be auto generated instead?

Reply:
id is an auto-increment column, MySQL picks the next sequence number when 0 is inserted. I've added a comment to note that.

Reply:
Thanks! I wasn't aware of that.